### ISSUE TITLE
Add documentation on how to upload files

### ISSUE THREAD

**Issue by vincentdavis** [pyscript/pyscript#1050]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other feature requests and couldn't find a duplicate (including also the ``type-feature`` tag)
- [X] I confirmed that it's not related to another project area (see the above section)

### What is the idea?

Example of how to upload a file, maybe to load into pandas and display.
I think how this is done has changed recently as I have tried to follow many guides.
 I have spent alot of time trying to select a CSV file, upload it to local browser storage or stream it into a pandas DataFrame.

edit: these docs could be in the form of a tutorial or example to the pyscript-collective. (Or something else!)

### Why is this needed

Helpful example for data science users

**Comment by marimeireles** [comment]
Hey @vincentdavis thanks for opening the issue.
I don't think we have any examples or tutorials on how to upload files, even though that'd be a cool addition! (I'll edit your issue to reflect this idea)
But you can check this project by: https://github.com/dylanjcastillo/pandas-cheatsheet/
Where he does something very similar :)
Hope it helps a bit!

**Comment by marimeireles** [comment]
Besides this example you might find this piece by Jeff more comprehensible: https://jeff.glass/post/pyscript-image-upload/
It's a bit old (alpha release of pyscript) but it's another point of view.

**Comment by vincentdavis** [comment]
@marimeireles Thanks for the suggestions. I have spent way too much time on this :-)
My challenge is that while I am ok python,  data science dev, I have no applied knowledge of HTML / javascript. The examples in the pyscript repo are all too _fancy_ and don't include comments explaining the steps or include lots of other libraries. I got something working that uses panel but I get stuck when I try to do more.

I am trying right now to implement a simple HTML-based `<input type="file"...` pass the file to a python function, show some text about the file and provide a download of the modified file. Hints please :-)

Here is my current implementation but I am not happy with it. 
[Fit File Info](https://github.com/vincentdavis/VAM_WAM_cheaters/blob/main/Fit_File_Info.html)

**Comment by WebReflection** [comment]
@vincentdavis sorry for being late in here but [this file](https://github.com/vincentdavis/VAM_WAM_cheaters/blob/main/web.py) doesn't use comments at all neither so it's hard to follow ... have you tried latest PyScript, as everything changed in the meanwhile and maybe it just works now?

**Comment by WebReflection** [comment]
This is going to be documented and live pretty soon so I am working ahead of time here, closing as we covered file uploads to the virtual FS and files download from it.
